### PR TITLE
chore: move templates to this repo

### PR DESCRIPTION
**Pull request by michalmrazek**
## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: templates/blank/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+pip install wanna-ml
+```
+
+# Launch to cutter
+```
+wanna init --template blank
+```
+or without wanna, manually
+```
+cookiecutter https://github.com/avast/wanna-blank-cookiecutter
```
Comment (racinmat) [inline templates/blank/README.md:19]: I guess this should be updated?

File: templates/blank/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+```
+
+# Answer the following question, values will be used to in your wanna-ml config
+
+```
+project_name [project_name]: 
+project_owner_fullname [project owner]: 
+project_owner_email [you@avast.com]: 
+project_version [0.0.0]: 
+project_description [Link to WANNA project page on CML]:
```
Comment (racinmat) [inline templates/blank/README.md:29]: do we want to mention CML in OSS project?

File: templates/blank/{{cookiecutter.project_slug}}/mkdocs.yaml [hunk 3]
```diff
@@ -0,0 +1,64 @@
+site_name: {{ cookiecutter.project_name }}
+repo_url: https://git.int.avast.com/{{ cookiecutter.project_repo_name }}
```
Comment (racinmat) [inline templates/blank/{{cookiecutter.project_slug}}/mkdocs.yaml:2]: I guess this should be changed so we're not mixing the internal and public git servers?

File: templates/blank/{{cookiecutter.project_slug}}/pyproject.toml [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+extend-ignore = "E203"
+max-line-length = 120
+
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core>=1.0.0"]
+
+[[tool.poetry.source]]
+name = "avast-repo-local" # This name will be used in the configuration to retrieve the proper credentials
+url = "https://artifactory.ida.avast.com/artifactory/api/pypi/pypi-local/simple/" # URL used to download your packages from
```
Comment (racinmat) [inline templates/blank/{{cookiecutter.project_slug}}/pyproject.toml:85]: I'm not sure if this should be public

File: templates/blank/{{cookiecutter.project_slug}}/pyproject.toml [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+[[tool.poetry.source]]
+default = true 
+name = "avast-repo-remote" # This name will be used in the configuration to retrieve the proper credentials
+url = "https://artifactory.ida.avast.com/artifactory/api/pypi/pypi-remote/simple/" # URL used to download your packages from
```
Comment (racinmat) [inline templates/blank/{{cookiecutter.project_slug}}/pyproject.toml:90]: I'm not sure if this should be public

**Review by racinmat — CHANGES_REQUESTED** [review]

File: templates/sklearn/{{cookiecutter.project_slug}}/pyproject.toml [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+taskipy = "^1.10.1"
+
+[tool.taskipy.tasks]
+black = "black pipeline"
+check = "task isort && task black && task pylint"
+isort = "isort pipeline"
+pylint = "pylint pipeline"
+
+[tool.pytest.ini_options]
+addopts = "--verbose --pylint --pylint-rcfile=pyproject.toml --cov={{cookiecutter.project_slug}} --cov-report html --mypy"
```
Comment (racinmat) [inline templates/sklearn/{{cookiecutter.project_slug}}/pyproject.toml:31]: do we want to run black here? or is it ran together with pylint automatically?

**Review by jsilva — APPROVED** [review]

File: templates/sklearn/{{cookiecutter.project_slug}}/pyproject.toml (hunk 6 above)
Comment (michalmrazek) [reply templates/sklearn/{{cookiecutter.project_slug}}/pyproject.toml:31]: Black is also run, but not part of pytest :)

**Pull request merged**